notebooks/notebooks/paper/win-rate-vs-board-size.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    import utils
</pre><p>nit: should be grouped on a separate line since it's a local import</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>